tokio/tests/time_interval.rs

```diff
@@ -209,3 +209,76 @@ fn poll_next(interval: &mut task::Spawn<time::Interval>) -> Poll<Instant> {
 fn ms(n: u64) -> Duration {
     Duration::from_millis(n)
 }
+
+mod tmp_tests {
+    use std::{
+        pin::Pin,
+        task::{Context, Poll},
+        time::Instant,
+    };
```

Comment:
We don't use this import style in Tokio.
```suggestion
    use std::pin::Pin;
    use std::task::{Context, Poll};
    use std::time::Instant;
```

Reply:
Yes, we don't currently use it.

As an aside, I wish we did and would take a PR that did a big pass-through of Tokio to update all imports... though others should chime in before anyone does the work.

Reply:
Ah OK thanks for letting me know - updated.